py/desispec/scripts/daily_processing.py

```diff
@@ -443,6 +446,12 @@ def daily_processing_manager(specprod=None, exp_table_path=None, proc_table_path
                 write_table(ptable, tablename=proc_table_pathname)
             if override_night is None or continue_looping_debug:
                 sleep_and_report(10, message_suffix=f"after updating queue information", dry_run=dry_run)
+        if run_only_once:
+            print("Exiting after running only once")
```

Comment:
Could we please make this a bit more explicit; "looping only once" or something even more explicit than that?

Reply:
Done.